<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertWindow</class>
 <widget class="QMainWindow" name="ConvertWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>325</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>517</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ENVI Conversion tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/artwork/convert.svg</normaloff>:/artwork/convert.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="inpGain">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="outputFile"/>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btnConvert">
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>82</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Gain</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btnCaptureBrowse">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Source folder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="captureFolder"/>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="btnBatch">
      <property name="text">
       <string>Batch</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="normalize">
      <property name="text">
       <string>Normalize</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btnNumpyBrowser">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Output file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Convert various images</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionNPY"/>
    <addaction name="actionNPZ"/>
   </widget>
   <addaction name="menuFormat"/>
  </widget>
  <action name="actionNPY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NPY</string>
   </property>
  </action>
  <action name="actionNPZ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NPZ</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
